src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -24,18 +24,49 @@ def import_from_package
           end
         end
 
-        redirect_to kiwi_image_repositories_path(package.kiwi_image)
+        redirect_to kiwi_image_path(package.kiwi_image)
       end
 
       def show
+        @repositories = @image.repositories.order(:order)
+        @package = @image.package
+        @project = @package.project
+      end
+
+      def edit
+        @repositories = @image.repositories.order(:order)
+        @package = @image.package
+        @project = @package.project
+      end
+
+      def update
+        ::Kiwi::Image.transaction do
+          @image.update_attributes!(image_params)
+          @image.write_to_backend
+        end
+        redirect_to action: :show
+      rescue ActiveRecord::RecordInvalid, Timeout::Error => e
+        flash[:error] = "Cannot update repositories for kiwi image: #{@image.errors.full_messages.to_sentence} #{e.message}"
+        redirect_back(fallback_location: root_path)
+      end
+
+      def is_outdated
```

Comment:
I agree, the naming is a bit strange. And even outside of a controller the method name would usually be 'outdated?'